### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mapra99**
Requesting Second code review. The first code review can be found on the pull request from the branch named "Main-branch". We created this new branch with a more adequate name, following the suggestion made by the first reviewer.

### CODE REVIEW

**Comment by Ispirett** [comment]
Really Awesome work folks :1st_place_medal:  You are 95% there. Final changes
- [x] Add the section under the Create Account button that starts with **By clicking Create account, you agree to our** and this section **Invisible reCAPTCHA by Google Privacy Policy and Terms of Use.**

When finished please submit for another review request.

**Comment by mapra99** [comment]
Added Terms of Use section below the submit button and in the footer of the fieldset.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (bolah2009) [inline index.html:2]: `Awesome`

Nice job using `lang="en"` (or variant) to the `html` start tag, this is good for internationalisation. You should check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics).  :+1: :star2:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                <a href="#" class="serviceIcon"><img src="images/quickIcon.png" alt="Quickbooks Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/mintIcon.png" alt="Mint Logo"></a>
+            </div>
+        </div>
+    </header>
+
+    <main>
+
+        <section>
+            <form name="create-account">
```
Comment (bolah2009) [inline index.html:41]: `AWESOME` 

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                    </button>
+                </div>
+            </div>
+            <div class="logo-container center-div">
+                <a href="#"><img src="images/intuit-logo.png" alt="Intuit Logo"></a>
+            </div>
+            <div class="services-container center-div">
+                <a href="#" class="serviceIcon"><img src="images/turboIcon.png" alt="Turbotax Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/quickIcon.png" alt="Quickbooks Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/mintIcon.png" alt="Mint Logo"></a>
```
Comment (bolah2009) [inline index.html:33]: `AWESOME`

Nice job, :clap: :star2:  adding an `alt` attribute to the `` element, this is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                        </div>
+                    </div>
+                    <br>
+
+
+                    <label for="email">
+                        Email address
+                    </label>
+                    <div class="input-cont">
+                        <input type="email" name="email" id="email" placeholder="You'll verify this later" required>
```
Comment (bolah2009) [inline index.html:57]: `AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

**Review by bolah2009 — APPROVED** [review]
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

I'm impressed and proud of your progress, you should be too :clap:

Good luck with your next project
Keep it up :star2:

Cheers
Bola

**Comment by mapra99** [comment]
Review approved. Merging to master

**Pull request merged**